weblate/trans/widgets.py

```diff
@@ -349,6 +349,15 @@ def render(self, response):
             "Kurinto Sans", Pango.Weight.NORMAL, 11, 0, f"  {percent_text}  "
         )[0].width
 
+        # Check for the 'badge_case' URL parameter and customize capitalization accordingly
+        badge_case = response.request.GET.get(
+            "badge_case", "lower"
+        )  # Default to lowercase
+
+        if badge_case == "upper":
```

Comment:
I'd rather use `capitalize` here, `upper` is something different:

```suggestion
        if badge_case == "capitalize":
```